.eslintrc.js

```diff
@@ -1,34 +1,32 @@
 module.exports = {
   env: {
     browser: true,
-    es6: true
+    es6: true,
   },
-  extends: ["plugin:vue/recommended", 'prettier', 'plugin:prettier/recommended' ],
+  extends: ["plugin:vue/recommended", "prettier", "plugin:prettier/recommended"],
   rules: {
-    'max-len': 'off',
-    'no-console': 'warn',
-    'no-return-await': 'warn',
-    'no-throw-literal': 'warn',
-    'vue/require-default-prop': 'off',
-    'vue/prop-name-casing': 'off',
-    'object-curly-spacing': 'off',
-    'lines-between-class-members': [
-      'warn',
-      'always',
-      {exceptAfterSingleLine: true}
+    "max-len": "off",
+    "no-console": "warn",
+    "no-return-await": "warn",
+    "no-throw-literal": "warn",
+    "vue/require-default-prop": "off",
+    "vue/prop-name-casing": "off",
+    // "vue/no-undef-properties": "warn",
```

Comment:
Since we've decided to not use no-undef-properties due to the large number of false positives, please remove it from here as well as all the `eslint-disable-next-line vue/no-undef-properties` that were introduced.